base/install_motor.sh

```diff
@@ -34,3 +33,45 @@ cd ..
 
 make -j${JOBS}
 make clean
+
+#Add pmac module
+
+cd ..
+
+git clone --depth 1 --branch ${PMAC_VERSION} \
+    https://github.com/dls-controls/pmac
+
+cd pmac
+
+rm -rf .git/ .git*
```

Comment:
`rm -rf .git*` will do the job for `.git` as well. `*` may expand to empty string and it will still match.